src/lib/formats/sdf_dsk.cpp

```diff
@@ -0,0 +1,202 @@
+// license:BSD-3-Clause
+// copyright-holders:Wilbert Pol, tim lindner
+/*********************************************************************
+
+    formats/sdf_dsk.h
+
+    SDF disk images. Format created by Darren Atkinson for use with
+    his CoCoSDC floppy disk emulator.
+    
+    http://cocosdc.blogspot.com/p/sd-card-socket-sd-card-socket-is-push.html
+
+*********************************************************************/
+
+#include <assert.h>
+
+#include "sdf_dsk.h"
+
+
+sdf_format::sdf_format()
+{
+}
+
+
+const char *sdf_format::name() const
+{
+	return "sdf";
+}
+
+
+const char *sdf_format::description() const
+{
+	return "SDF disk image";
+}
+
+
+const char *sdf_format::extensions() const
+{
+	return "sdf";
+}
+
+
+int sdf_format::identify(io_generic *io, uint32_t form_factor)
+{
+	const int header_size = 512;
```

Comment:
You should use `constexpr` rather than `const` for things that should always be evaluated at compile time.  Also, we encourage naming compile-time constants in `ALL_CAPS` for consistency.